### ISSUE TITLE
TXT output format

### ISSUE THREAD

**Issue by alfonsocaponi** [hahwul/dalfox#53]
Running dalfox on a remote xterm (with putty client, so without gui) on Ubuntu 18.4 we have a issue with TXT output format:

...
^M^[[37m[*] ^[[0mStart static analysis.. xx
^M^[[37m[*] ^[[0mStart parameter analysis.. xx
^M^[[37m[*] ^[[0mGenerate XSS payload and optimization.Optimization.. xx
...

**Comment by bsysop** [comment]
This "strange" characters is because dalfox put colors in output to make easier to visualization, maybe in future would have a "-n" or "-nc" to let user decide if want or not have colorized screen.

In any case you can remove it by now using:
```
sed 's/\x1b\[[^\x1b]*m//g' YOUR_FILE.txt
```

**Comment by hahwul** [comment]
@alfonsocaponi  @bsysop 
Thank you for your opinion! I'm aware of that, but I'm still thinking about how to deal with it. I'll patch it before this weekend!

**Comment by hahwul** [comment]
Hi @alfonsocaponi , @bsysop 
I fixed this issue, please update dalfox `v1.0.3-dev`.

```
go get -u github.com/hahwul/dalfox
```
![1414](https://user-images.githubusercontent.com/13212227/80689486-62b13000-8b08-11ea-8a0a-2622699ab56a.png)

**Comment by hahwul** [comment]
Oh, the data is a little different in the image  from 123 file, but it's because I did another test before taking a screenshot. Keep that in mind!

If there is a wrong move, please re-open it!
